build-time-analytics/src/test/java/io/quarkus/ts/buildtimeanalytics/AnalyticsUtils.java

```diff
@@ -0,0 +1,125 @@
+            "quarkus-jackson",
+            "quarkus-jaxb",
+            "quarkus-jdbc-mysql",
+            "quarkus-jdbc-postgresql",
+            "quarkus-jsonb",
+            "quarkus-jsonp",
+            "quarkus-kafka-client",
+            "quarkus-micrometer",
+            "quarkus-micrometer-registry-prometheus",
+            "quarkus-narayana-jta",
+            "quarkus-oidc",
+            "quarkus-openshift-client",
+            "quarkus-quartz",
+            "quarkus-reactive-pg-client",
+            "quarkus-reactive-routes",
+            "quarkus-rest-client",
+            "quarkus-rest-client-jaxb",
+            "quarkus-resteasy",
+            "quarkus-resteasy-jackson",
+            "quarkus-resteasy-jaxb",
+            "quarkus-resteasy-jsonb",
+            "quarkus-scheduler",
+            "quarkus-smallrye-graphql-client",
+            "quarkus-smallrye-reactive-streams-operators",
+            "quarkus-spring-boot-properties",
+            "quarkus-spring-data-jpa",
+            "quarkus-spring-di",
+            "quarkus-spring-security",
+            "quarkus-undertow",
+            "quarkus-vertx",
+    };
+    public static final String[] EXTENSION_SET_B = new String[] {
+            "quarkus-avro",
+            "quarkus-container-image-openshift",
+            "quarkus-hibernate-orm-rest-data-panache",
+            "quarkus-jaxrs-client-reactive",
+            "quarkus-jdbc-mariadb",
+            "quarkus-jdbc-mssql",
+            "quarkus-mutiny",
+            "quarkus-oidc-client",
+            "quarkus-oidc-client-reactive-filter",
+            "quarkus-qute",
+            "quarkus-reactive-mysql-client",
+            "quarkus-rest-client-reactive",
+            "quarkus-resteasy-reactive",
+            "quarkus-resteasy-reactive-jackson",
+            "quarkus-smallrye-context-propagation",
+            "quarkus-smallrye-fault-tolerance",
+            "quarkus-smallrye-health",
+            "quarkus-smallrye-jwt",
+            "quarkus-smallrye-jwt-build",
+            "quarkus-smallrye-metrics",
+            "quarkus-smallrye-openapi",
+            "quarkus-smallrye-opentracing",
+            "quarkus-smallrye-reactive-messaging",
+            "quarkus-smallrye-reactive-messaging-kafka",
+            "quarkus-spring-cache",
+            "quarkus-spring-cloud-config-client",
+            "quarkus-spring-data-rest",
+            "quarkus-spring-scheduled",
+            "quarkus-spring-web",
+            "quarkus-websockets",
+            "quarkus-websockets-client",
+    };
+
+    public static String getNonExcludedAppName() {
+        return getAppName(NON_EXCLUDED_GROUP_ID);
+    }
+
+    public static String getExcludedAppName() {
+        return getAppName(EXCLUDED_GROUP_ID);
+    }
+
+    private static String getAppName(String groupId) {
+        return String.format("%s:app", groupId);
+    }
+
+    public static QuarkusCliClient.CreateApplicationRequest getCreateAppRequest() {
```

Comment:
It returns an instance of class named `CreateApplicationRequest`. `createApp()` sounds like it would create the app.

Reply:
Then "appBuilder()" maybe?Or, since the request is only used to add extensions, may be we should just add "extensions" parameter to "whenCreate" method in AbstractAnalticsIT?